include/acutest.h

```diff
@@ -1629,6 +1638,36 @@ acutest_is_tracer_present_(void)
 }
 #endif
 
+#ifdef ACUTEST_MACOS_
+static bool
+AmIBeingDebugged(void)
```

Comment:
Please rename the function to have the prefix `acutest_` as the rest of identifiers.

It's too prevent a name clash with people's code using `acutest.h`. Especially if the function is copied from some Apple materials, the name clash is likely a real concern.

Reply:
@mity thanks for the feedback! 🙂 I've addressed this in https://github.com/mity/acutest/pull/61/commits/e20cf63d4d0c16eebd77e790d4be01e3dece9bfb.